### ISSUE TITLE
Issue updating WordPress with A2 Optimized active

### ISSUE THREAD

**Issue by brashrebel** [A2-Hosting/a2-optimized-wp#8]
Just tested on [this site](https://thewildnorth.com) to update from 4.7.5 to 4.8 and the core update page loaded and stopped. No activity, no update, the page was gray and empty. Troubleshot for a while and ended up getting the update to succeed only after disabling A2 Optimized.

**Comment by andymelichar** [comment]
Thanks for the report, Kyle - couple questions:
- Did you by chance try re-enabling A2 Opt after the core upgrade?
- Did you have W3TC/Caching enabled as part of A2 Opt before the upgrade?

**Comment by brashrebel** [comment]
Yes and yes. I tried a few other things that made no difference like updating other plugins and disabling other plugins. It just loaded an empty admin screen with nothing on the page where one would usually see the lines detailing the update process and then redirecting to the new version's welcome page.

I was going to try disabling W3TC next but A2O resulted in the update succeeding so I went no further. I did enable it again after the update and everything seems normal.
